web/public/templates/dojos/events/session-details.dust

```diff
@@ -22,6 +22,10 @@
       </tr>
     </tbody>
   </table>
+  <div>
+    <h4> Notes: </h4>
```

Comment:
We'll also need to use the `i18n` key here and on the next line e.g. [this line](https://github.com/josmas/cp-zen-platform/blob/application_notes%23564/web/public/templates/dojos/events/session-details.dust#L8)